src/script/api/game_changelog.hpp

```diff
@@ -78,6 +78,9 @@
  * API removals:
  * \li GSError::ERR_PRECONDITION_TOO_MANY_PARAMETERS, that error is never returned anymore.
  *
+ * Other changes:
+ * \li GSTile::DemolishTile takes one extra parameter to enable magic bulldozer behaviour
```

Comment:
Not true anymore, but the new mode should be listed in API additions instead.

Reply:
Oops that was embarrassing. Thanks for the catch.